xml/cap_depl_ldap.xml

```diff
@@ -0,0 +1,162 @@
+    </para>
+<screen>&prompt.user;uaac curl /identity-providers?rawConfig=true -X POST \
+    -k \
+    -H 'Content-Type: application/json' \
+    -H 'X-Identity-Zone-Subdomain: scf' \
+    -d '{
+  "type" : "ldap",
+  "config" : {
+    "ldapProfileFile" : "ldap/ldap-search-and-bind.xml",
+    "baseUrl" : "<replaceable>ldap://ldap.example.com:389</replaceable>",
+    "bindUserDn" : "<replaceable>cn=admin,dc=example,dc=com</replaceable>",
+    "bindPassword" : "<replaceable>password</replaceable>",
+    "userSearchBase" : "<replaceable>dc=example,dc=com</replaceable>",
+    "userSearchFilter" : "<replaceable>uid</replaceable>={0}",
+    "ldapGroupFile" : "ldap/ldap-groups-map-to-scopes.xml",
+    "groupSearchBase" : "<replaceable>dc=example,dc=com</replaceable>",
+    "groupSearchFilter" : "<replaceable>member={0}</replaceable>"
+  },
+  "originKey" : "ldap",
+  "name" : "<replaceable>My LDAP Server"</replaceable>,
+  "active" : true
+  }'
+</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Verify the LDAP identify provider has been created in the <literal>scf</literal> zone. The output should now contain an entry for the <literal>ldap</literal> type.
+    </para>
+<screen>&prompt.user;uaac curl /identity-providers -k -H "X-Identity-Zone-Id: scf"</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Use the &cfcli; to target your &productname; deployment.
+    </para>
+<screen>&prompt.user;cf api --skip-ssl-validation https://api.example.com</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Login as an administrator.
+    </para>
+<screen>&prompt.user;cf login
+API endpoint: https://api.example.com
+
+Email> admin
+
+Password>
+Authenticating...
+OK
+</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Create users associated with your LDAP identity provider.
+    </para>
+<screen>&prompt.user;cf create-user <replaceable>username</replaceable> --origin ldap
+Creating user username...
+OK
+
+TIP: Assign roles with 'cf set-org-role' and 'cf set-space-role'.
+</screen>
+   </listitem>
+   <listitem>
+    <para>
+     Assign the user a role. Roles define the permissions a user has for a given org or space and a user can be assigned  multiple roles. See <link xlink:href="https://docs.cloudfoundry.org/concepts/roles.html">Orgs, Spaces, Roles, and Permissions</link> for available roles and their corresponding permissions. The following example assumes that an org named <replaceable>Org</replaceable> and a space named <replaceable>Space</replaceable> have already been created.
+    </para>
+<screen>&prompt.user;cf set-space-role <replaceable>username</replaceable> <replaceable>Org</replaceable> <replaceable>Space</replaceable> <replaceable>SpaceDeveloper</replaceable>
+Assigning role RoleSpaceDeveloper to user username in org Org / space Space as admin...
```

Comment:
Hmm, `RoleSpaceDeveloper`, not `SpaceDeveloper` (as specified on the command line)?

Reply:
I just tested and `set-space-role` seems to append `Role` to the output regardless of which role is specified. `set-org-role` doesn't seem to do this.

Reply:
Ah, right; I guess it's just the CLI being weird.